src/popup.html

```diff
@@ -50,19 +50,43 @@ <h1 class="settings-hl i18n-content" data-i18n-message-id="popupSettingsTitle"><
       <main-panel class="signed-in-panel hidden">
         <p class="fx-relay-font-met aliases-remaining remaining-plural"></p>
         <onboarding-panel>
-          <img class="onboarding-img">
-          <h1 class="onboarding-h1"></h1>
-          <p class="onboarding-p"></p>
+          <!--Non Premium Panel-->
+          <div class="content-wrapper">
+            <img class="onboarding-img">
+            <h1 class="onboarding-h1"></h1>
+            <p class="onboarding-p"></p>
 
-          <div class="onboarding-pagination">
-            <button class="panel-nav previous-panel i18n-attribute" data-direction="-1" data-i18n-attribute="aria-label" data-i18n-message-id="popupOnboardingShowPrevious" aria-label=""></button>
-            <span class="panel-num fx-relay-font-met">
-              <span class="current-panel"></span>/
-                <span class="total-panels">3</span>
-            </span>
-            <button class="panel-nav next-panel i18n-attribute" data-direction="1" data-i18n-attribute="aria-label" data-i18n-message-id="popupOnboardingShowNext" aria-label=""></button>
+            <div class="onboarding-pagination">
+              <button class="panel-nav previous-panel i18n-attribute" data-direction="-1" data-i18n-attribute="aria-label" data-i18n-message-id="popupOnboardingShowPrevious" aria-label=""></button>
+              <span class="panel-num fx-relay-font-met">
+                <span class="current-panel"></span>/
+                  <span class="total-panels">3</span>
+              </span>
+              <button class="panel-nav next-panel i18n-attribute" data-direction="1" data-i18n-attribute="aria-label" data-i18n-message-id="popupOnboardingShowNext" aria-label=""></button>
+            </div>
           </div>
-        </onboarding-panel>
+          <!-- Premium Panel-->
+          <div class="premium-wrapper">
+            <div class="premium-dashboard">
+              <ul>
+                <li class="dashboard-info">
+                  Email aliases used
```

Comment:
Does this text need to be pulled from `browser.i18n.getMessage()`?

Reply:
Yup, I added that into my WIP bit.